client/src/main/java/com/alibaba/nacos/client/config/NacosConfigService.java

```diff
@@ -93,7 +93,7 @@ private void initNamespace(Properties properties) {
                 System.getProperty(SystemPropertyKeyConst.IS_USE_CLOUD_NAMESPACE_PARSING,
                     String.valueOf(Constants.DEFAULT_USE_CLOUD_NAMESPACE_PARSING)));
 
-        if (Boolean.valueOf(isUseCloudNamespaceParsing)) {
+        if (Boolean.parseBoolean(isUseCloudNamespaceParsing)) {
```

Comment:
```java
    /**
     * Returns a {@code Boolean} with a value represented by the
     * specified string.  The {@code Boolean} returned represents a
     * true value if the string argument is not {@code null}
     * and is equal, ignoring case, to the string {@code "true"}.
     *
     * @param   s   a string.
     * @return  the {@code Boolean} value represented by the string.
     */
    public static Boolean valueOf(String s) {
        return parseBoolean(s) ? TRUE : FALSE;
    }
```
From the JDK, these two methods seem to be identical?

Reply:
Yeah, but there was an extra three-wood operation

Reply:
You lose efficiency, even if it's just a little bit

Reply:
OK, I think these two methods also have different return types. ```parseBoolean``` returns native type which doesn't need an extra unboxing operation.

Reply:
I didn't notice that. Now I've learned. Thank you